CMakeLists.txt

```diff
@@ -134,6 +134,8 @@ target_include_directories(${PROJECT_NAME}
     ${GFLAGS_INCLUDE_DIRS}
     ${GLOG_INCLUDE_DIRS}
     ${GTSAM_INCLUDE_DIR}
+    ${OPENGV_INCLUDE_DIR}
+    ${DBoW2_INCLUDE_DIRS}
```

Comment:
@ruffsl I don't see why you need to put DBoW2 include dirs here, this should be already done when calling DBoW2::DBoW2 in target_link_libraries (are you using DBoW2 headers through Kimera?).

Reply:
I don't think that interface is working as you expect. My hunch is that the project was relying on and on finding the headers via the system global paths given the sketchy `*sudo* make install`

https://github.com/MIT-SPARK/Kimera-VIO/blob/a913c97d2fbc3310166448776fa60451583c35ff/Dockerfile_18_04#L68-L74

or depended on a erroneous limitation in legacy catkin for non-isolated devel space builds:

https://answers.ros.org/question/320613/catkin_make-vs-catkin_make_isolated-which-is-preferred/?answer=320706#post-id-320706

When using current build tools, each package must explicitly declare it's install steps given the use of sanitary/isolated builds. Thus every downstream package must similarly explicitly import the resources they need, as no cmake context it implicitly shared across workspace packages.